PersonApi/Startup.cs

```diff
@@ -91,26 +91,6 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddSwaggerGen(c =>
             {
-                c.AddSecurityDefinition("Token",
```

Comment:
I think this might be used within the swagger UI to allow authorization when trying out the endpoints. I'm not sure if this is sufficient to expose the Swagger UI itself.

Reply:
I don't there is any other place that has the Swagger authorization in place unless you've spotted it somewhere?

Reply:
I'm pretty sure that any changes will need to be in the ApiGateway, as that is where the request is validated for the presence of a valid token.

Reply:
Mirela, has spoken to me about the changes that need to take place, I'm going to make this PR has a draft as I'm still working on it